server/src/cache/macros.rs

```diff
@@ -0,0 +1,4 @@
+//! log macro's for cache logging
+
+use lib_common::log_macros;
+log_macros!("cache", "backend::cache");
```

Comment:
This results in log messages in the namespace `backend::cache::cache`
Should be changed to `log_macros!("cache", "backend");` which will override the default `app` prefix with `backend`

Reply:
https://github.com/aetheric-oss/svc-cargo/pull/50/commits/41a7901b5736914cff12cc50c1defb0864305bc6